neo4j-jdbc/src/main/java/org/neo4j/jdbc/DefaultTransactionImpl.java

```diff
@@ -172,13 +164,18 @@ private <T> T execute(CompletableFuture<T> future, int timeout) throws SQLExcept
 		}
 		catch (ExecutionException ex) {
 			var cause = ex.getCause();
+			if (cause == null) {
+				cause = ex;
+			}
+			var sqlException = new SQLException("An error occurred while handling request.", cause);
```

Comment:
```suggestion
			var sqlException = new SQLException("An error occurred while handling request", cause);
```

No dots at the end of the message (in line with JDK and Spring messages)

Reply:
Updated for this class.